CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 As a minor extension, we also keep a semantic version for the `UNRELEASED`
 changes.
 
+## [0.10.0]
```

Comment:
```suggestion
## [0.10.0] - UNRELEASED
```

According to https://github.com/input-output-hk/hydra/blob/master/CONTRIBUTING.md#versioning--changelog